tests/core/manager/test_response_formatters.py

```diff
@@ -39,6 +39,7 @@
         "code": -32601,
         "message": "the method eth_getTransactionByHash does not exist/is not "
         "available",
+        "data": MethodUnavailable("method not found"),
```

Comment:
This is the actual response from a client (in this case Geth) that doesn't implement a method. The `data` key doesn't get returned, so we can't rely on that to determine which error to throw. We'll want to make this test pass without the `data` key.